packages/core/src/tooltip/TooltipBase.tsx

```diff
@@ -0,0 +1,62 @@
+import { StatusIndicator, ValidationStatus } from "../status-indicator";
+import { FloatingArrow, FloatingArrowProps } from "@floating-ui/react";
+import { TooltipProps } from "./Tooltip";
+import { makePrefixer } from "../utils";
+import { useFormFieldProps } from "../form-field-context";
+
+import tooltipCss from "./Tooltip.css";
```

Comment:
Should this be renamed to `TooltipBase`? Or what's the best way to ensure css injection can be detected by linter?

Reply:
I don't think there's a specific rule around what the import should be named. The linter is just making sure that the css is assigned to a variable at the moment